doc/work_management/jira.rst

```diff
@@ -70,19 +80,29 @@ Task Process
     
         * If task is a fixed length (i.e. training/conferences/etc.) create one task (see task creation above).
         * Fixed length tasks can be larger than 5 Story Points.
+        * In case a non-fixed length task needs to be larger than 5 Story Points, discuss with the Sprint Lead or manager.
+        * Short tasks may be 0.5 Story Point.
```

Comment:
Short tasks should not be shorter that 0.5 SP.  It is preferred that all tasks are in whole number story points.

Reply:
Good suggestion. I'd like to emphasize that only 0.5 is allowed if necessary and any integer value up to 5 SP.